index.js

```diff
@@ -0,0 +1,3 @@
+const subtract = (a, b) => {
```

Comment:
Are you sure this is the best naming convention for this function?

Reply:
Good catch, will fix it, thanks!